modules/panorama/variables.tf

```diff
@@ -64,6 +64,12 @@ variable "enable_plan" {
   type        = bool
 }
 
+variable "panorama_disk_type" {
+  description = "Specifies the type of managed disk to create. Possible values are either Standard_LRS, StandardSSD_LRS, Premium_LRS or UltraSSD_LRS."
```

Comment:
Maybe add validation? Some of `string` values in Azure are case sensitive (especially since provider v. 3.x), so it would make sense to check if the spelling is correct by verifying if the value is one of  `Standard_LRS`, `StandardSSD_LRS`, `Premium_LRS` or `UltraSSD_LRS`.

Reply:
Good catch, I add that :)